pkg/image/validation/image.go

```diff
@@ -0,0 +1,85 @@
+package validation
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+const (
+	imageComponent = "image"
+)
+
+var validImageTypes = []string{image.TypeISO, image.TypeRAW}
+var validArchTypes = []string{string(image.ArchTypeARM), string(image.ArchTypeX86)}
```

Comment:
Can we move these to the `validateImage()` function or do we expect to reuse them?

Reply:
I don't have aspirations of reusing them. My mentality is that I tend to make lists like this outside of the code path to prevent re-initializing the list on each pass through. However, seeing as there's only going to be one pass through the validation, that concern isn't an issue. I'll move them because I like the idea of limiting the scope of them.